### ISSUE TITLE
Issue with docker-gcplogs-driver - Viewing your logs section

### ISSUE THREAD

**Issue by ghost** [GoogleCloudPlatform/community#220]
Issue Description
Viewing your logs needs to be updated. It's not under Global anymore but under "GCE VM Instance"

![image](https://user-images.githubusercontent.com/11959532/29165195-3ffb11e4-7dc2-11e7-9f2d-1892488e96f3.png)

**Comment by ToddKopriva** [comment]
@tswast , can you have a look at this bug against a tutorial that you contributed?

**Comment by tswast** [comment]
Sounds like we just need to update the screenshots in https://cloud.google.com/community/tutorials/docker-gcplogs-driver

I'd want to re-run through the tutorial to verify, though. It's likely something else has changed since then.

**Comment by ToddKopriva** [comment]
Thanks, Tim.

**Comment by tswast** [comment]
Just ran the tutorial from scratch.

Still appears to be under "global" for me.

![image](https://user-images.githubusercontent.com/247555/73861606-e1951600-4802-11ea-89ad-db79b02179de.png)

Not seeing "gcplogs-docker-driver" under GCE VM Instance.

![image](https://user-images.githubusercontent.com/247555/73861683-fa053080-4802-11ea-8958-1680176970df.png)

I am finding a few links that need to be updated, so I'll send a PR soon with those.

**Comment by ToddKopriva** [comment]
Thanks again, @tswast .

**Comment by lgandras** [comment]
This bug exists indeed:

### Reproduction steps:
1. Create a machine with boot disk Container Optimized OS
2. Execute `docker run -d --name mysite --log-driver=gcplogs -p 80:80 nginx`
3. Execute `curl localhost`
4. Reset the VM
5. Execute `docker run -d --name mysite2 --log-driver=gcplogs -p 80:80 nginx`
6. Execute `curl localhost`

### Expected behaviour:
You get two entries under "GCE VM Instance, _instance-name_" > "gcplogs-docker-driver"

### Actual behaviour:
You get one entry under "Global" > "gcplogs-docker-driver"
You get one entry under "GCE VM Instance, _instance-name_" > "gcplogs-docker-driver"

### Additional information:
In the actual behaviour, there's a single difference between the two entries. The first one is missing the labels *instance_id* and *zone*:

```
resource:
    labels:
        project_id: "my-project-id"
```

whereas the second one (after a reset) looks like this:

```
resource:
    labels:
        instance_id: "2836207941276610115"    
        project_id: "my-project-id"    
        zone: "europe-west1-b" 
```

### Bottom line
The actual behaviour can be explained as the *gcplogs driver not attaching* the _instance id_ nor the _zone_ at the initial startup and then, after a reset, attaching the missing labels and thus, logging the entry correctly under the *GCE VM Instance* logs with the respective instance name.

The first entry doesn't have these labels, and that's the reason Stackdriver fills these entries under *Global*.

**Comment by tswast** [comment]
@lgandras Could you file that with the Docker folks? This repo only addresses the actual tutorial text itself

**Comment by lgandras** [comment]
I did, thanks: https://github.com/moby/moby/issues/40505
